src/main/java/puke/command/TodoCommand.java

```diff
@@ -0,0 +1,77 @@
+package puke.command;
```

Comment:
Instead of putting Todo, Deadline and Event under command do you think it would be better to put them under another category called tasks